### PR TITLE
Feature/bulk upload validations

### PR DESCRIPTION
**Pull request by pjshumphreys**
https://trello.com/c/n8oB9pyL

### CODE REVIEW

**Comment by pjshumphreys** [comment]
[Bulk Upload Validations](https://trello.com/c/n8oB9pyL/53-bulk-upload-validations)

File: server/models/BulkImport/csv/establishments.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const STOP_VALIDATING_ON = ['UNCHECKED', 'DELETE', 'NOCHANGE'];
 const localAuthorityEmployerTypes = [1, 3];
 const nonDirectCareJobRoles = [1, 2, 4, 5, 7, 8, 9, 13, 14, 15, 17, 18, 19, 21, 22, 23, 24, 26, 27, 28];
 const permanantContractStatusId = 1;
+const notHeadOfficeOrCqcRegulated = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,/* 16, (16 = Head office service code, which should be excluded) */ 17, 18];
```
Comment (aaron-russell) [inline server/models/BulkImport/csv/establishments.js]: Wouldn't the comment be better above the array rather than in the middle of it?

Comment (pjshumphreys) [reply server/models/BulkImport/csv/establishments.js]: I specifically put it here to be in context of the array. The array is sorted so I wanted to make sure that it's clear that it's intentional that 16 isn't there and why

**Comment by aaron-russell** [comment]
Happy with reply

**Pull request merged**